### ISSUE TITLE
Provide means to change the language of BibleTime via the GUI

### ISSUE THREAD

**Issue by jaakristioja** [bibletime/bibletime#1]
Moved from Bugzilla.

The GUI should provide means to change the GUI language either immediately at runtime or offer a restart option to change the language.

**Comment by mgruner** [comment]
I would disagree. Almost no program I know offers such an option, usually they use the user language of the environment. I'll close this for now, feel free to reopen if there is a strong use case. Let's concentrate on important stuff.

**Comment by jaakristioja** [comment]
Reopening due to #192 and personal motivation.

**Comment by jaakristioja** [comment]
While there is still a long way to go to properly implement this, I've always had this issue in mind while refactoring various parts of BibleTime. So this issue has actually received quite a lot of attention even while being closed.

**Comment by m-cat** [comment]
BibleTime isn't picking the system language (English) for me on OSX. It seems to be randomly picking from among my keyboard languages, but I'm not sure what its exact logic is. I didn't open a new bug report because I don't feel like it.

**Comment by jaakristioja** [comment]
This issue is still relevant, as users need or want to change the language of the GUI at runtime. I just received another support request via e-mail and remembered that it would also help with issues like #358.
